include/cbreader/rmesh.h

```diff
@@ -12,7 +12,7 @@ constexpr std::uint32_t MAX_SOUND_EMITTERS = 16;
 struct Header
 {
 	// Length prefixed string
-	std::uint32_t header_length = 0;
+	//std::uint32_t headerLength = 0;
```

Comment:
(nit) I suggest removing the commented out lengths. I mostly added them just because they were apart of the struct itself. But I'm all for removing them, escaped my mind that stream.read could be passed as an argument.